### PR TITLE
Console cleanup

### PR DESCRIPTION
**Pull request by Ran73Cohen**
#1017

### CODE REVIEW

**Comment by Ran73Cohen** [comment]
@Velua Can we remove all console.timer?
Can you look at things thrown from console.warn and see if we can repair warnings we throw,
You can just remove some of them if they are irreverent to track.

Since the console is more clean now I will look into some of the deprecation warnings we get.

File: bumpVersion.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ const newVersionString = [major, minor, lastVersion].join(".");
 const newString = raw.replace(versionString, newVersionString);
 
 fs.writeFileSync("./package.json", newString);
-console.log("Bumped package.json version to", newVersionString);
```
Comment (Velua) [inline bumpVersion.js:18]: Keep this one, doesn't actually run in the app but as a script.

File: src/components/common/DataTable.vue [hunk 2]
```diff
@@ -176,10 +176,6 @@ export default class DataTable extends BaseComponent {
     const endIndex = this.currentPage * perPage;
     const startIndex = endIndex - perPage;
     const items = this.sortedItems.slice(startIndex, endIndex);
-    const itemsWithoutId = items.filter(x => !x.id);
-    if(itemsWithoutId.length > 0) {
-      console.log(itemsWithoutId, "are without an ID");
-    }
```
Comment (Velua) [inline src/components/common/DataTable.vue:182]: Keep this, handy for to find any bugs. will only run if there is a bug.

**Review by Velua — APPROVED** [review]

**Comment by Velua** [comment]
Yeah kill the ```console.time```s as well.

**Review by Velua — APPROVED** [review]

**Pull request merged**